cvmfs/catalog_mgr_impl.h

```diff
@@ -841,6 +844,43 @@ bool AbstractCatalogManager<CatalogT>::IsAttached(const PathString &root_path,
 }
 
 
+template <class CatalogT>
+void AbstractCatalogManager<CatalogT>::StageNestedCatalog(
```

Comment:
i was wondering if we could combine it with `findcatalog` or `mountsubtree` but i think this is not possible.  especially the `if (MountSubtree(path, best_fit, true /* is_listable */, NULL)) {` seems interesting but i guess you get too muhc latency in case the `mountsubtree` should fail